app/src/main/java/com/github/libretube/compat/PendingIntentCompat.kt

```diff
@@ -1,20 +1,30 @@
 package com.github.libretube.compat
 
 import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
 import android.os.Build
 
+// TODO: Use AndroidX's PendingIntentCompat class instead once it becomes available.
```

Comment:
Have there been any announcements for a `PendingIntentCompat` being added to the `androidx` library in the near future recently? (just out of interest)

Reply:
It's there in the repo: https://github.com/androidx/androidx/blob/androidx-main/core/core/src/main/java/androidx/core/app/PendingIntentCompat.java

Reply:
So it's probably going to be added with the next version bump of `androidx.core`?

Reply:
Seems like it.